src/main/java/com/ericsson/ei/mongo/MongoDBHandler.java

```diff
@@ -52,6 +52,7 @@
 import lombok.Getter;
 import lombok.Setter;
 
+@SuppressWarnings("deprecation")
```

Comment:
Please do not add SuppressWarnings. A warning is an indication that something must be updated, Suppressing it makes it possible to forget that there is an issue and in the future this could break EI.

I will instead create an issue about fixing this warning.